packages/components/psammead-bulletin/src/__snapshots__/index.test.jsx.snap

```diff
@@ -988,6 +988,266 @@ exports[`Bulletin should render live video correctly 1`] = `
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding-right: 0.5rem;
+}
+
+.c8 > svg {
+  color: #FFFFFF;
+  fill: currentColor;
+  width: 1.0625rem;
+  height: 1rem;
+  margin: 0;
+}
+
+.c7 {
+  background-color: #222222;
+  border: 0.0625rem solid transparent;
+  color: #FFFFFF;
+  padding: 0.75rem;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+@media (min-width:37.5rem) {
+  .c1 {
+    width: 50%;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c1 {
+    padding: 0;
+  }
+}
+
+@supports (grid-template-columns:fit-content(200px)) {
+  .c1 {
+    width: initial;
+    grid-column: 1 / span 6;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c3 {
+    width: 50%;
+    padding-left: 1rem;
+  }
+}
+
+@supports (grid-template-columns:fit-content(200px)) {
+  .c3 {
+    width: initial;
+    grid-column: 1 / span 6;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    padding: 1rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c4 {
+    font-size: 1.125rem;
+    line-height: 1.375rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    padding: 0 0 0.5rem 0;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c7 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c7 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c7 {
+    display: -webkit-inline-box;
+    display: -webkit-inline-flex;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    padding: 0.5rem 1rem;
+  }
+}
+
+<div
+  class="c0"
+>
+  <div
+    class="c1"
+  >
+    <img
+      alt="Iron man"
+      class="c2"
+      src="https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg"
+    />
+  </div>
+  <div
+    class="c3"
+    dir="ltr"
+  >
+    <h3
+      class="c4"
+      dir="ltr"
+    >
+      <a
+        class="c5"
+        href="https://bbc.co.uk"
+      >
+        <span
+          role="text"
+        >
+          <span
+            class="c6"
+            lang="en-GB"
+          >
+            Listen, 
+          </span>
+          <span>
+            This is the headline
+          </span>
+        </span>
+      </a>
+    </h3>
+    <div
+      aria-hidden="true"
+      class="c7"
+      dir="ltr"
+    >
+      <span
+        class="c8"
+        dir="ltr"
```

Comment:
Correct me if I'm wrong but, I was expecting to see an `svg` inside this span

Reply:
Good catch, there was a mistake with a bulletin type ("radio" instead of "audio"). It is now fixed